lib/ui/mail_view.dart

```diff
@@ -7,11 +7,28 @@ import 'package:summer2022/ui/bottom_app_bar.dart';
 import 'package:summer2022/ui/top_app_bar.dart';
 
 class MailViewWidget extends StatelessWidget {
-  final List<MailPiece> mailPieces = List.generate(
-      10,
-          (index) =>
-          new MailPiece("", "", DateTime.now(), "John Doe", "Lorem ipsum dolor sit amet, ", "")
-  );
+
+  //this is temporary for list view display, need to eventually delete when search results are achieved
+  List<MailPiece> mailPieces = _createMailPieces();
+
+  static List<MailPiece> _createMailPieces() {
+    List<MailPiece> _mailPieces = List.generate(
```

Comment:
I think we can remove the dummy data now since we are actually pulling and storing results

Reply:
Agreed, it was already there for the general list, and I made the extra item to make a specific mailpiece based on a specific item from a specific email, just to make sure it works.  Mike said I didn't need to work on displaying the search results list, though I can help with that, but I believe that is part of searching as you said.